ci/prow/make_config.go

```diff
@@ -322,28 +328,117 @@ func combineSlices(a1 []string, a2 []string) []string {
 	return res
 }
 
-// Consolidate whitelisted and skipped branches with newly added whitelisted/skipped
+// intersectSlices returns intersect of 2 slices
+func intersectSlices(a1 []string, a2 []string) []string {
+	var res []string
+	s1 := sets.NewString(a1...)
+	for _, e2 := range a2 {
+		if s1.Has(e2) {
+			res = append(res, e2)
+		}
+	}
+	return res
+}
+
+// exclusiveSlices returns elements in a1 but not in a2
+func exclusiveSlices(a1 []string, a2 []string) []string {
+	var res []string
+	s2 := sets.NewString(a2...)
+	for _, e1 := range a1 {
+		if !s2.Has(e1) {
+			res = append(res, e1)
+		}
+	}
+	return res
+}
+
+// Get go113 image name from base image name, following the contract of
+// [IMAGE]:[DIGEST]-> [IMAGE]-go113:[DIGEST]
+func getGo113ImageName(name string) string {
+	go113ID := "-go113"
```

Comment:
This should be a constant.

Reply:
Trying not to make const considering how large this file is, created a function instead